frontend/include/chpl/framework/ErrorMessage.h

```diff
@@ -159,23 +85,15 @@ class ErrorMessage final {
   inline Location location() const { return location_; }
 
   inline bool operator==(const ErrorMessage& other) const {
-    return isDefaultConstructed_ == other.isDefaultConstructed_ &&
-           kind_ == other.kind_ &&
-           id_ == other.id_ &&
-           location_ == other.location_ &&
-           message_ == other.message_ &&
+    return kind_ == other.kind_ && id_ == other.id_ &&
+           location_ == other.location_ && message_ == other.message_ &&
```

Comment:
I (weakly) preferred these on the same line. Is this clang-format talking, or is this change deliberate?

Reply:
It's `clang-format`. By the same line do you mean individual lines, as they were before? That's how I prefer it, strongly enough to change it back. If you meant same line as in all on one line, that would go over 80 char limit.

Reply:
Sorry, I meant on "different lines", as before.